src/app/market/buy/checkout-process/checkout-process.component.ts

```diff
@@ -397,3 +402,7 @@ export class CheckoutProcessComponent implements OnInit, OnDestroy {
   }
 
 }
+
+export enum errorType {
```

Comment:
I don't think this needs to be exported, since its only used in this component. Also, to fix the warnings, it might be helpful to declare this near the top of the file instead of at the bottom.